CHANGELOG.md

```diff
@@ -12,10 +12,15 @@
 
 * Fixes a bug in the login `uponSubmit` filter where a user could login without meeting the requirement.
 
+### Fixes
+
+* Resolve inline image URLs correctly when in edit mode and not in the default locale.
```

Comment:
It looks like one of Miro's PRs got caught up in this PR somehow. But I'm going to ignore it because I know I approved it.